### ISSUE TITLE
Can we use libraries, C libraries, machine learning?

### ISSUE THREAD

**Issue by CryptoKR** [touilleMan/godot-python#183]
Are we able to use libraries like numpy, scipy, tensorflow, keras, etc..

**Comment by apockill** [comment]
Yes, you definitely can. I have [a project](https://github.com/apockill/CreepyCrawlAI/tree/master) right now that uses tensorflow, numpy, and tf-agents. 

I think there's a better way of installing things (I can't seem to figure out how to use the pip that comes prepackaged with the build), but I've been installing dependencies by doing:

 ```shell
pip install --upgrade -r requirements.txt --target pythonscript/x11-64-cpython/lib/python3.6/site-packages
```

Now this only works if the pip you are using is for the same python version as the one shipped with godot-python. 

Does anyone else have better instructions for dependency installation?

**Comment by CryptoKR** [comment]
Thanks for the info. We should have a doc that instructs us how to use libraries. I got conflicting info when I was searching for this answer on forums like reddit.

**Comment by touilleMan** [comment]
@apockill you seem to have a cool project here ;-)
What version of Godot/godot-python do you use ?
In the next release of godot-python we will ship CPython in a much better way (see #181), hence using pip would be just a matter of doing `./pythonscript/x11-64/bin/python -m pip` (and yes @CryptoKR  you are able to install anything from pypi \o/)

**Comment by apockill** [comment]
@touilleMan thank you for godot-python! I'm hoping to be more active helping folks on the issues. 

> What version of Godot/godot-python do you use ?

Currently I'm still using the v0.11.1 build that was one-click installable via the assetstore. I experimented with building v0.20.1, but I was unable to get it to run, and debugging was difficult (this may be related to some of the stdout issues I saw recently). 

> In the next release of godot-python we will ship CPython in a much better way (see #181), hence using pip would be just a matter of doing ./pythonscript/x11-64/bin/python -m pip 

I'm excited to try it out! It seems like the build system has been getting better recently. Hopefully that means more updates on the assetstore :grin:
